aerosol_dynamics/physical_parameters.py

```diff
@@ -0,0 +1,33 @@
+"""
+Centralized location for physical parameters.

```

Comment:
This is good for now, but I think some things (temp, viscosity, mean free path) will be removed from this when we make an 'air parcel' class for the particles and gases to be in.

Reply:
Aktwo and I started working on an Environment class that would attempt to address this. Temp/Pressure would move there, and I guess we would put mean free path there to be calculated from environment properties. I can address that in a separate issue.

Reply:
@aktwo @wkchuang it would be good to make that public whenever you feel comfortable --- either in your forks or in a remote branch here. Up to you though :)